### DISCUSSION TITLE
Trying to figure out what is the best time to run the "grab" for epgs to set cron job and does grab merge the epgs?

### DISCUSSION THREAD

**Discussion by DimSumDev** [iptv-org/community#615] [Q&A] [answered]
I notice when running the "grab" it grabs it for the next day and not current day. (for my time zone CST)

I'm trying to set a cron job on my server so I can pull the epg every 3 days (seems 3 days of programming and my 'maxConnections` settings causes the least errors and is fairly stable.

So the question is, for U.S. CST what is the best time to schedule these jobs? is it 11:59 pm or should I got for a certain UTC time?

Also, does the `npm grab` keep or is there a way to merge copies so I have a day back and the new 3 day pull or should I just pull a full week?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
It all depends on the source that you are using and amount of data you are grabbing. Provide source and numbers of channels so that others can share their experience. 

For most people, it comes down to trial and error.
